docs/reference/accelerometer.md

```diff
@@ -112,6 +120,7 @@ wb_accelerometer_enable(tag, sampling_period)
 wb_accelerometer_disable(tag)
 period = wb_accelerometer_get_sampling_period(tag)
 [x y z] = wb_accelerometer_get_values(tag)
+lookup_table = wb_accelerometer_get_lookup_table(tag)
```

Comment:
We should somehow show that this is an array (see previous line).

Reply:
I am not sure if I understand. Something like this?
```matlab
lt = wb_accelerometer_get_lookup_table(tag)
lookup_table = array2table([ lt(1:3:end); lt(2:3:end); lt(3:3:end) ]', 'VariableNames', {'Input', 'Output', 'Std'})
```

Reply:
Oh no I was thinking about something much simpler, such as:
`[value0 value1 ...] = wb_accelerometer_get_lookup_table(tag)`

Reply:
It seems that the example above doesn't work, with:
```
val = wb_accelerometer_get_values(acc)
[x y z] = wb_accelerometer_get_values(acc)
```
I get:
```
Error using wb_accelerometer_get_values
Too many output arguments.
Error in matlab (line 45)
[x y z] = wb_accelerometer_get_values(acc)
Error in launcher (line 161)
  eval(WEBOTS_CONTROLLER_NAME);
val =
    0.0007    0.0078    9.8506
```
So, `val = wb_accelerometer_get_values(acc)` returns correct result, but another line fails.

Reply:
Hum..
In that case it is indeed better to keep it simple, maybe we can just write:
```
lookup_table_array = wb_accelerometer_get_lookup_table(tag)
```
Just to emphasis that this is an array.

Reply:
Ok, I will do that. But now I intrigued by this:
```
[x y z] = wb_accelerometer_get_values(acc)
```
It seems it doesn't work and it is written in many places. Should I do something about that?

Reply:
Yes if you want to fix it gladly (but please do this in another PR so that it doesn't mix with the changes specific to this PR).
Or if you prefer not to do this right now feel free to open an issue about it and you (or someone else) will fix it later.

Reply:
Yes, we should probably fix it by writing something like:
```matlab
x_y_z_array = wb_accelerometer_get_values(acc)
```